src/routes/album/album.component.tsx

```diff
@@ -303,6 +357,21 @@ const Album = ({
                   </PopoverWindow>
                 )}
               </div>
+
+              {user?.id && (
```

Comment:
same as above. I'd prefer to see it in separate component with its own subscription to store. 
I mean, you can create a new file and select currentRate here, to don't update entire album component, when its value will change